crates/core_arch/src/x86/sse.rs

```diff
@@ -1470,6 +1470,7 @@ pub unsafe fn _mm_sfence() {
 /// For more info see [`_mm_setcsr`](fn._mm_setcsr.html)
 ///
 /// [Intel's documentation](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_getcsr)
+#[rustc_deprecated(since = "1.38.0", reason = "modifying MXCSR causes undefined behavior in LLVM, see https://github.com/rust-lang/stdarch/issues/781")]
```

Comment:
If this is merged into rustc, it should hit stable at version 1.40.0, so please use that as since version.

Reply:
Thanks, fixed now